src/rules/__tests__/form-control-has-label.test.ts

```diff
@@ -5,8 +5,22 @@ makeRuleTester("form-control-has-label", rule, {
   valid: [
     "<label for=''><input type='text' /></label>",
```

Comment:
Not related to the original issue I had files, but this line doesn't pass HTML validation:
1. An empty string is not allowed for `for`
2. If label has a `for` attribute, the nested form control (input). must have an id that matches the value of `for`.

It's also questionable if a label without text should pass.